Source/LinuxParking.API/Controllers/ParkingController.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using AutoMapper;
+using LinuxParking.API.Domain.Interfaces.Services;
+using LinuxParking.API.Domain.Models;
+using LinuxParking.API.Domain.Resources;
+using LinuxParking.API.Extensions;
+using LinuxParking.API.Utils;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LinuxParking.API.Controllers
+{
+    [Route("/api/station/{stationId}/spot/{spotId}/[controller]")]
+    public class ParkingController : Controller
+    {
+        private readonly IParkingService _parkingService;
+        private readonly IMapper _mapper;
+
+        public ParkingController(IParkingService parkingService, IMapper mapper)
+        {
+            _parkingService = parkingService;
+            _mapper = mapper;
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> CreateAsync([FromRoute] int stationId, [FromRoute] int spotId, [FromBody] CreateParkingResource resource)
```

Comment:
Considering  GetAsync and DeleteAsync methods are named similar to HttpGet and HttpDelete, it would look nice if this method was named PostAsync instead!